s2sml/torch_s2s_dataset.py

```diff
@@ -277,13 +275,18 @@ def zscore_values(self, mnv, stdv):
         # if mean and standard deviation are NOT provided do this (only era5)
         if mnv == None or stdv == None:
         
-            self.mean_val = xr.open_mfdataset(
-                self.list_of_era5, concat_dim='sample', combine='nested')[var].mean(
-                skipna=True).values
-
-            self.std_val = xr.open_mfdataset(
-                self.list_of_era5, concat_dim='sample', combine='nested')[var].std(
-                skipna=True).values
+            tmp = xr.open_mfdataset(self.list_of_era5, concat_dim='sample', combine='nested')[var]
+            tmp = self.box_cutter(tmp)
+            
+            if not self.norm_pixel:
+                
+                self.mean_val = tmp.mean(skipna=True).values
+                self.std_val = tmp.std(skipna=True).values
```

Comment:
Question about how the standard deviation is calculated - is it over space and time?

Reply:
This is space and time, yeah.

Reply:
yup! space and time!

Reply:
Thanks - so no weighting needed for the standard deviation calculation right? (I'm guessing no, but also trying to wrap my head around this for another project...)

Reply:
(note, that it is only across space and time for the time range selected, to avoid training data leakage)

Reply:
hmmm... I think we need to do weighting for std as well if we do it for mean

Reply:
I was going to say the same as @mariajmolina, but I'm googling to double check.